README.md

```diff
@@ -1,5 +1,5 @@
-[![Build Status](https://travis-ci.org/OCA/sale-reporting.svg?branch=8.0)](https://travis-ci.org/OCA/sale-reporting)
-[![Coverage Status](https://coveralls.io/repos/OCA/sale-reporting/badge.png?branch=8.0)](https://coveralls.io/r/OCA/sale-reporting?branch=8.0)
+[![Build Status](https://travis-ci.org/OCA/sale-reporting.svg?branch=9.0)](https://travis-ci.org/OCA/sale-reporting)
+[![Coverage Status](https://coveralls.io/repos/OCA/sale-reporting/badge.png?branch=9.0)](https://coveralls.io/r/OCA/sale-reporting?branch=9.0)
```

Comment:
yes, it's weird, it was added in the very first one (dfebab3d225b9eb0156afbcdeefa62059238dac3). Just revert the change  :scissors: